Parsers/Cisco_ISEParser.txt

```diff
@@ -0,0 +1,43 @@
+// KQL Parser for Cisco ISE (Identity Services Engine) Syslog
+// Last Updated Date: Sept 20, 2020
+//
+// Parser Notes: 
+// 1. Extracts common fields based on Regex
+//      TimeStamp, hostname, ise_type, sessionkey, Segments,Segnumber
+// 2. Common Json Property bag to hold all the remaining key value pairs. 
+//     - (Easily extractable using mv-expand and parse_json on demand)
+//     - Some of the keys are duplicate in logs hence we can`t use directly make_bag, using make_list. 
+// 3. Regular expression in this parser assumes field names in alphanumeric format (a-zA-Z0-9- ) and values in alphanumeric 
+//      and some special characters format (a-zA-Z0-9-_:/@.#{};= ) which covers most cases per initial testing. 
+//      To include any additional special chars modify regex accordingly.
+// 4. Parser expects Syslog to be of below format in the same order and in a field - RawData. 
+//    For any changes, adjust the regex accordingly or submit an issue of Github with the format you are trying.
+//    Expected Format - (SYSLOGTIMESTAMP:timestamp} {HOSTNAME:hostname} {DATA:cisco_ise_type} {INT:session_key} {INT:total_segments} {INT:seg_number} %{GREEDYDATA:message- key-value pairs}
+// 5. Reference - Documentation Cisco ISE PSN Logs Remote Format: 
+//  https://www.cisco.com/c/en/us/td/docs/security/ise/2-6/Cisco_ISE_Syslogs/Cisco_ISE_Syslogs/Cisco_ISE_Syslogs_chapter_00.html#id_101580
```

Comment:
Remove the locale specific info in the URL.

Reply:
It seems Cisco does not support global locale like docs.microsoft.com . I tried removing en/us but throws 404.

Have added instruction in the notes, if user wants to change.. Also build is passing so i think we can keep this URL.

Reply:
Yeah, the check for locale is not done on parsers, but good practice to have no locale any where that we can.  Understood that Cisco does not have it available.